cogs/helpers.py

```diff
@@ -35,15 +33,20 @@
 import numpy as np
 
 # Other utilities
+import os
 import re
 import math
 import time
-import os
 import datetime
+import pickle
+import feedparser
 from .utils.paginator import Pages
 from .utils.requests import fetch
 
 MCGILL_EXAM_URL = "https://www.mcgill.ca/exams/dates"
+
+CFIA_FEED_URL = "http://inspection.gc.ca/eng/1388422350443/1388422374046.xml"
```

Comment:
Should this URL be configurable? The hash code or whatever these numbers are make me kind of nervous that the feed changes regularly....

Reply:
It doesn't change, they just have multiple feeds for allergen contamination and location, but I picked the nationwide feed for food recalls.